enduro.toml

```diff
@@ -64,6 +64,24 @@ bucket = "aips"
 [a3m]
 address = "127.0.0.1:7000"
 shareDir = "/home/a3m/.local/share/a3m/share"
+name = "mytest"
```

Comment:
I think this should be removed, as the app name doesn't need to be configurable.